jobs/future-effective-filings/utils/__init__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright © 2019 Province of British Columbia
```

Comment:
These don't seem to be used. Also, why not just use isoformat() ?

Reply:
you are right! They don't get used. The structure of the folder was a copy over from the other job(s) and this was  dangling code